app/utils/channel/index.ts

```diff
@@ -118,3 +118,17 @@ export function generateChannelNameFromDisplayName(displayName: string) {
     }
     return name;
 }
+
+export function compareNotifyProps(propsA: Partial<ChannelNotifyProps>, propsB: Partial<ChannelNotifyProps>): boolean {
+    if (
```

Comment:
Wouldn't it be better to just do something like:
`return A.x === B.x && A.y === B.y...`

Reply:
I have changed the implementation for this feature and I'm not using this function anymore atm.  However, I'm going to keep it until I work on the Settings feature as it may come in handy.  If not, I'll remove it.